modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -81,7 +81,7 @@ interface OuterContainerApis {
 interface ToolbarApis {
   readonly setGroups: (toolbar: AlloyComponent, groups: SketchSpec[]) => void;
   readonly refresh: (toolbar: AlloyComponent) => void;
-  readonly toggle?: (toolbar: AlloyComponent) => void;
+  readonly toggle?: (toolbar: AlloyComponent, options?: { skipFocus: boolean }) => void;
```

Comment:
Generally, alloy has avoided variable arguments, and relied on boulder etc. to standardise. This one is a bit different, though, as it's not just the spec coming in, but an argument to an API.

I think many of my issues with defaulted arguments stem from pre-Typescript days, where it made it much more likely that a typo wouldn't be picked up. However, I _think_ I'd still prefer to see something more like: 

`toggle`
`toggleWithoutFocusing`

And then _internally_, they can use an argument passed to a common function, but the APIs would be separate. I don't really know if I can justify that, though ... and the naming I have chosen there is horrible. Just something to think about.

Reply:
I am not sure I can understand which problem this API splitting can help us solve. :(

Reply:
Primarily just that alloy doesn't really use defaulted parameters anywhere. At least it didn't before. But it's up to you. It's mainly a thing of consistency.

Reply:
@ephox-mogran Removed the `options` parameter from alloy components in favor for `toggle` and `toggleWithoutFocusing` functions.